### PR TITLE
Add PC Console Changes from ARR

### PR DESCRIPTION
**Pull request by haroldrubio**
This PR contains some changes I've made to the ARR PC consoles that may be useful to other conferences in the CSV exports

### CODE REVIEW

File: openreview/conference/templates/programchairWebfield.js [hunk 1]
```diff
@@ -518,12 +518,14 @@ var getUserProfiles = function(userIds, reviewerBidCounts, areaChairBidCounts, a
         profileMap[profile.id] = {
           id: profile.id,
           name: name,
+          gender: profile.content.gender,
```
Comment (melisabok) [inline openreview/conference/templates/programchairWebfield.js:521]: I'm not sure about exporting this data in all the PC consoles. I know this data is public but we don't show in the public profile so the owner of the profile doesn't know.

**Review by melisabok — APPROVED** [review]

**Comment by xkopenreview** [comment]
i think the change for ac meta-review denominator issue in overview tab is missing?

**Comment by xkopenreview** [comment]
discussed this with @haroldrubio. Harold will make a pr in web to port the changes in this pr over to pc console component.

**Comment by melisabok** [comment]
@xkopenreview can this be merged?

**Review by xkopenreview — APPROVED** [review]

**Pull request merged**